setup.cfg.template

```diff
@@ -81,10 +75,9 @@
 #
 # The Agg, Ps, Pdf and SVG backends do not require external dependencies. Do
 # not choose MacOSX, or TkAgg if you have disabled the relevant extension
-# modules.  Agg will be used by default.
+# modules.  The default is determined by fallback.
```

Comment:
I don't understand this change.  Isn't the fallback approach being removed by this PR?  (Or is this referring to something else recent that I probably missed while away from the project for the last couple of years...)

Reply:
The fallback has been moved to runtime (not build time).